package.json

```diff
@@ -22,7 +22,7 @@
 	},
 	"scripts": {
 		"analyze": "polymer analyze --input cosmoz-*.js  > analysis.json",
-		"lint": "eslint --cache --ext .js,.html . && polymer lint cosmoz-*.js",
+		"lint": "eslint --cache cosmoz-*.js && polymer lint cosmoz-*.js",
```

Comment:
I don't think this change is necessary. I would leave eslint to also run against the demo and test `html` files.

Reply:
Yes true, didn't think of that!